frontend/src/app/features/member/form/form-validators.ts

```diff
@@ -0,0 +1,48 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
```

Comment:
can we move this whole class to a global helper package? Since member is most likely not the only place where we want to handle dates?